src/assets/google-analytics.js

```diff
@@ -1,6 +1,21 @@
-(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+function initGoogleAnalytics(i, s, o, g, r, a, m) {
+  i['GoogleAnalyticsObject'] = r;
+  i[r] = i[r] || function () {
+      (i[r].q = i[r].q || []).push(arguments)
+    }, i[r].l = 1 * new Date();
+  a = s.createElement(o),
+    m = s.getElementsByTagName(o)[0];
+  a.async = 1;
+  a.src = g;
+  m.parentNode.insertBefore(a, m);
+}
 
-ga('create', 'UA-164824-57', 'auto');
+if (window.location.host.match(/prx.org/)) {
+  initGoogleAnalytics(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+
+  ga('create', 'UA-164824-57', 'auto');
+} else if (window.location.host.match(/prx.tech/)) {
+  initGoogleAnalytics(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+
+  ga('create', 'UA-164824-58', 'auto');
+}
```

Comment:
Looking for documentation on what 'none' as the cookieDomain actually does...  https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#cookieDomain I assume 'none' is for when you are using localhost rather than faking a domain like we do with pow. Guess I could just try it. Thought we didn't really want this locally at all though. I don't want to take up another of our 11 Google Analytics properties we have left for our local dev environments, so just re-use the staging id as long as it's not actually posting page views or events.